fmts/testdata/pylint.in

```diff
@@ -0,0 +1,44 @@
+************* Module torch.nn.functional
+pytorch/torch/nn/functional.py:874:4: W0622: Redefining built-in 'input' (redefined-builtin)
+pytorch/torch/nn/functional.py:878:4: C0200: Consider using enumerate instead of iterating with range and len (consider-using-enumerate)
+pytorch/torch/nn/functional.py:878:8: C0103: Variable name "d" doesn't conform to snake_case naming style (invalid-name)
+pytorch/torch/nn/functional.py:887:16: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
+pytorch/torch/nn/functional.py:892:12: C0103: Variable name "d" doesn't conform to snake_case naming style (invalid-name)
+pytorch/torch/nn/functional.py:897:20: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
+pytorch/torch/nn/functional.py:907:4: W0622: Redefining built-in 'input' (redefined-builtin)
+pytorch/torch/nn/functional.py:906:0: R0913: Too many arguments (6/5) (too-many-arguments)
+pytorch/torch/nn/functional.py:939:11: I1101: Module 'torch._C' has no '_nn' member, but source is unavailable. Consider adding this module to extension-pkg-allow-list if you want to perform analysis based on run-time introspection of living objects. (c-extension-no-member)
+pytorch/torch/nn/functional.py:939:11: W0212: Access to a protected member _nn of a client class (protected-access)
+pytorch/torch/nn/functional.py:939:11: W0212: Access to a protected member _C of a client class (protected-access)
+pytorch/torch/nn/functional.py:943:4: W0622: Redefining built-in 'input' (redefined-builtin)
+pytorch/torch/nn/functional.py:942:0: R0913: Too many arguments (6/5) (too-many-arguments)
+pytorch/torch/nn/functional.py:971:11: I1101: Module 'torch._C' has no '_nn' member, but source is unavailable. Consider adding this module to extension-pkg-allow-list if you want to perform analysis based on run-time introspection of living objects. (c-extension-no-member)
+pytorch/torch/nn/functional.py:971:11: W0212: Access to a protected member _nn of a client class (protected-access)
+pytorch/torch/nn/functional.py:971:11: W0212: Access to a protected member _C of a client class (protected-access)
+pytorch/torch/nn/functional.py:975:4: W0622: Redefining built-in 'input' (redefined-builtin)
+pytorch/torch/nn/functional.py:974:0: R0913: Too many arguments (6/5) (too-many-arguments)
+pytorch/torch/nn/modules/fold.py:1:0: R0801: Similar lines in 2 files
+==torch.nn.intrinsic.qat.modules.conv_fused:[53:75]
```

Comment:
Shouldn't we include the bellow lines?

Reply:
I wasn’t sure what the matching format should be to fit it. Any suggestions?